emission/tests/modellingTests/TestSimilarityMetric.py

```diff
@@ -6,26 +6,93 @@ class TestSimilarityMetric(unittest.TestCase):
 
     def testODsAreSimilar(self):
         generate_points_thresh = 0.001  # approx. 111 meters
-        similarity_threshold = 500  # 
-        # random, but, points are sampled within a circle and should always be < sim threshold
-        trips = etmm.generate_mock_trips('bob', 2, [0, 0], [1, 1], threshold=generate_points_thresh)
+        similarity_threshold = 111  # 
```

Comment:
why did you have to change the threshold to 111? I can understand filtering for o and d, but why do you have to change the threshold?

Reply:
Same reason as https://github.com/e-mission/e-mission-server/pull/933/commits/710d1a5791212b540b883dedd5613a562071edc9#r1312355176 .

Reply:
given that we have now changed the mock trip creation code, this should still work with 500, correct?

Reply:
yes it does. 
And it is currently set to 500 in the latest commit.